Email templates! (#1657)
writethedocs/www: docs/conf/portland/2022/cfp_email_templates.rst

:template: {{year}}/generic.html
:orphan:

Speaker email templates
=======================

.. Make this whole file conditional

{% if cfp_variables['print_templates'] %}

.. contents::
    :local:
    :depth: 1
    :backlinks: none

CFP  - Acceptance template
~~~~~~~~~~~~~~~~~~~~~~~~~~

TBD

CFP - Rejection template
~~~~~~~~~~~~~~~~~~~~~~~~

TBD

CFP - Waitlist template
~~~~~~~~~~~~~~~~~~~~~~~

TBD

01 - Speaker logistics template
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Subject:
   WTD {{city}} {{year}} -- speaker logistics

::

   Hi folks!

   Just wanted to drop you all a quick note covering some logistics.

   * I've added some questions to our [CFP tool (Pretalx)]({{cfp.url}}) for you about your pronouns, interesting facts and name pronunciation. Please log in at {{cfp.url}} and answer those (although we'll only need them closer to the event).

   * If you haven't done so already, please upload a speaker pic to your Pretalx account, it'll look so much better than the anonymous outline.

>    * Private speaker slack channel! If you're not on the slack already, [join the WTD slack](https://join.slack.com/t/writethedocs/shared_invite/zt-tfokbfyb-4Yxwk8oTZGAQDHmITJGQyQ). Once you're signed up, or if you're already on there, ping me @plaindocs so I can add you to the private speaker channel. It contains all of our past speakers, who will be happy to offer advice or answer questions.

Reviewer: We should probably make this a variable in conf.py too because the link rotates periodically and if we have multiple copies, some will be left behind.

Author: Done :+1:

   ➡️ [book an audio-visual check]({{cfp_variables['calendly']}})

   Remember, all Q&A sessions **start at the end of your recording**, in the Speaker Q&A session (the Monday and Tuesday Q&A sessions have different URLs).

   Some speakers like to hang out in the event chat and answer questions during their talk, but that is entirely up to you. We'll be moderating questions regardless, and then the Q&A will be a live video call with the MC.

   If you have any questions at all about the event, you definitely know how to reach me by now.

   Sam

05 - Post conf
~~~~~~~~~~~~~~

Feedback:
   {{cfp_variables['feedback_form']}}
Gift:
   {{cfp_variables['speaker_gift']}}

----

Subject:
   WTD {{city}} {{year}} -- feedback, hoodies and THANKS

::

   Hey Folks,

   It is a wrap! Thank you one last time for your hard work, insight and creativity.

   We'll be publishing the videos at some point this week or next, keep an eye on Twitter, Slack or the mailing list for those. 

   We'd love to know how you found the whole process, from A to Z so we can improve next time. To that end we've got an anonymous (keep in mind that there aren't so many speaks) feedback form for you here:

>    ➡️ [Speaker feedback form]({{cfp_variables['feedback_form']}})

Reviewer: Feels like there are multiple styles in the emails for task lists. This emote, a normal list, a list with ascii checkboxes. Don't know if that's on purpose.

Author: Could probably use some tweaking yeah.

   Finally, as is tradition at our in person events, we'd love to send you a free WTD speaker hoodie and sketchnote print of your talk, please fill in the form before {{cfp.gifts_by}}.

   🎁 [Speaker gift form]({{cfp_variables['speaker_gift']}})

   Sam

{% else %}

> Populate the CFP environment variables to see the email templates.

Reviewer: ```suggestion
Populate the CFP environment variables to see the email templates, like so:

::

    export WTD_CFP_UPLOAD='TODO'
    export WTD_CFP_SPEAKER_TICKET='TODO'
    export WTD_CFP_CALENDLY='TODO'
    export WTD_CFP_FEEDBACK_FORM='TODO'
    export WTD_CFP_SPEAKER_GIFT='TODO'
```

   * In a change from our in person event, we'll be hosting moderated Q&A in a separate Hopin room, directly after each talk.

   Next week I'll send over a provisional schedule, and if having a particular slot would make it easier to attend the Q&A, let me know and I'll see what I can do.

   And while we're here, ;-) if you don't have a profile picture set in [Pretalx](https://pretalx.com/write-the-docs-{{shortcode}}-{{year}}/login/), now would be a great time to add one.

   And I think that is it! I'm excited to see this taking shape and excited to see all of your hard work on the virtual stage!

   Please get in touch if I can help with anything, if you have worries, thoughts or ideas.

   Sam

03 - Provisional schedule template
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Upload:
   {{cfp_variables['upload']}}

----

Subject:
   WTD {{city}} {{year}} -- schedule and upload drive

::

   Hi Folks,

   Hope you're all well!

   We're about five weeks out from the conference now, lots of stuff is happening behind the scenes, and I hope you're feeling good about recording.

>    We've got a [provisional schedule]({{cfp.preview}})) up, and I'd *love it* if you could check your time-slot and make sure you can do a live Q&A immediately after your talk is streamed. Note that the talk slots include the Q&A time. We'll make that public on **1st September**.

Author: Gah, missed this date

>    We've got a [provisional schedule]({{cfp.preview}})) up, and I'd *love it* if you could check your time-slot and make sure you can do a live Q&A immediately after your talk is streamed. Note that the talk slots include the Q&A time. We'll make that public on **1st September**.

Author: ```suggestion
   We've got a [provisional schedule]({{cfp.preview}})) up, and I'd *love it* if you could check your time-slot and make sure you can do a live Q&A immediately after your talk is streamed. Note that the talk slots include the Q&A time. We'll make that public in approximately one week.
```

>    A few folks asked where to upload talk recordings when you have them (by **cfp.video_by** right?):

Author: ```suggestion
   A few folks asked where to upload talk recordings when you have them (by **{{cfp.video_by}}** right?):
```

Reviewer: How did I overlook that 🙈

Author: Both of us! :see_no_evil:
